curvefs/util/build.sh

```diff
@@ -3,6 +3,25 @@
 # Copyright (C) 2021 Jingli Chen (Wine93), NetEase Inc.
 
 ############################  GLOBAL VARIABLES
+#get tag version
+tag_version=`git status | grep -w "HEAD detached at" | awk '{print $NF}' | awk -F"v" '{print $2}'`
+if [ -z ${tag_version} ]
+then
+    echo "not found version info, set version to 9.9.9"
+    tag_version=9.9.9
```

Comment:
do not use "9.9.9", use another mark or only commit id

Reply:
fix